doc/RecordingControl.xml

```diff
@@ -1684,8 +1755,193 @@ SetRecordingJobMode(JobToken, Active)
       </varlistentry>
     </variablelist>
   </appendix>
+  <appendix xml:id="_refCloudRecording">
+    <title>Object storage recording format (Normative)</title>
+    <section>
+      <title>Overview</title>
+      <para>
+        This annex describes the requirements for recording from a device to an object storage for later use by a receiver.
+        For this, the device uploads objects to the object storage.
+        The device could be a camera, the receiver could be a video player.
+        The object storage typically resides in the cloud or network.
+        <xref linkend="_refCloudRecordingDataFlow" /> shows the data flow for object storage recording.
+      </para>
+      <figure xml:id="_refCloudRecordingDataFlow">
+        <title>Data flow for object storage recording</title>
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="media/RecordingControl/cloud-recording-data-flow.svg" contentdepth="3.5em" />
+          </imageobject>
+        </mediaobject>
+      </figure>
+      <para>
```

Comment:
I feel like anyone that was not present at our meetings will have difficulty to comprehend this section of the specification. Should we instead attempt to define the keywords in terms of MP4 files? For example, a segment is an MP4 which contains at least 1 media track within, and must match the ISO specification as indicated by the tt:RecordingFormat
Then, a span consists of one or more segments of continuous recording, having no time gaps between each segment.

Reply:
@jflevesque-genetec Would the following be easier to understand in your opinion?

> The device uploads recorded media as objects containing fragmented MP4 data according to ISO/IEC 14496-12, the ISO base media file format.
> In this annex, a media object containing fragmented MP4 data is called segment.
> Each segment starts with a header containing initialization data, usually consisting of an `FileTypeBox` and `MovieBox` box.
> Each segment contains one or more fragments with the actual media, usually consisting of a `MovieFragmentBox` and `MediaDataBox` box.
> Each segment contains recorded media of a continuous period of time.
> For a single recording, multiple segments may contain recorded media of one point in time if video, audio, and metadata are written into seperate segments.
>
> Multiple segments form a span, containing recorded media of a continuous period of time.
> In this way, the size of segments can be limited when recording continuously for a longer period of time 
> For a single recording, only a single span contains recorded media of one point in time.

If you think this is better, I will also update the following sections slightly.

Reply:
This definition of "Span" still seems incomplete to me.
One or more "continuous" segments form a span, where continuity is determined by the monotonically increasing sequence number of each segment in addition to the absence of time between them.
This would cover the case of a camera recording continuously spans of 10 minutes for exemples. Or the fact that by deleting a segment in the middle of a span then you would effectively get two spans. What do you think?

Reply:
This is an informative introduction, so I don't think that we need to cover every case in this section, especially not what happens after the recording (we are only specifying the device in this annex). Also, it is hard to already cover the CMAF case in this introduction, where we have multiple independent sequence numbers. Maybe we should change the definition of a span, so we can have multiple spans covering a single point in time and each span (in the CMAF case) only contains a single type of media.